ReplayBrowser/Program.cs

```diff
@@ -50,5 +52,11 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
             {
                 webBuilder.UseKestrel();
                 webBuilder.UseStartup<Startup>();
+            })
+            .ConfigureServices(s => {
```

Comment:
Whats the reason for pulling this out of the startup? Not an issue, just curious.

Reply:
They depend on DbContext, but need setting up somewhere, and so they ended up outside. And put after dbcontext has migrated, so they don't error out as they rush to do things.

And migration has to be in other part of Startup because otherwise the EF dotnet tool just breaks in weird ways